README.md

```diff
@@ -246,11 +246,10 @@ Alternatively, if you only care about the version you're currently running, you
 
 # Deploying to PyPi
 
-1. Update the version number in `learnosity_sdk/_version.py`
-2. Install the dev requirements `pip install -r requirements-dev.txt`
-3. Run `python setup.py sdist` to create the source distribution
-4. Run `python setup.py bdist_wheel --universal` to create the binary distribution
-5. Run `twine upload dist/*` to deploy the distributions to PyPi
+1. Install the dev requirements `pip install -r requirements-dev.txt`
+2. Run `python setup.py sdist` to create the source distribution
+3. Run `python setup.py bdist_wheel --universal` to create the binary distribution
+4. Run `twine upload dist/*` to deploy the distributions to PyPi
```

Comment:
Ah, no. Literally just `make dist-upload`

Reply:
Instead of all four steps?

Reply:
Yup, it's one `Makefile` to rule them all

https://github.com/Learnosity/learnosity-sdk-python/blob/master/Makefile#L45-L58

Reply:
That, too https://github.com/Learnosity/learnosity-sdk-python/blob/master/Makefile#L21-L23